specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-08-15/mongorbac.json

```diff
@@ -0,0 +1,616 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB",
```

Comment:
This PR is created from 2022-05-15 (stable) version. The file exists in the preview version because the file originally created for preview release. I am not sure what do you mean by adding the file as previous commit

Reply:
> > ```
> >       "type": "boolean"
> > ```
> 
> same here
> 
> Refers to: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-08-15/cosmos-db.json:6255 in [b205f9f](https://github.com/Azure/azure-rest-api-specs/commit/b205f9f5c38a192992f0c0d3e5df95a007a2ed2a). [](commit_id = [b205f9f](https://github.com/Azure/azure-rest-api-specs/commit/b205f9f5c38a192992f0c0d3e5df95a007a2ed2a), deletion_comment = False)

this was always like that. Coming form the previous version of the spec.

Reply:
@raosuhas we're moving an API from public preview to GA/stable version, while keeping the public preview version as well at the same time. That's why the file exists in both the folders for now. There's no difference between the two files as well.

Reply:
The fist commit needs to be the previous version as explained here : https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version in step 7

by following this we can avoid unneccessary re-review of already checked in specs